master/reference/calicoctl/resources/bgpconfig.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BGP Configuration Resource (BGPConfiguration)
+---
+
+A BGP configuration resource (BGPConfiguration) represents BGP specific configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `bgpconfiguration`, `bgpconfig`, `bgpconfigurations`, `bgpconfigs`.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  logSeverityScreen: info
+  nodeToNodeMeshEnabled: true
+  asNumber: 63400
+```
+
+### BGP Peer Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Must be specified. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
+
+- Resource with the name "default" has a specific meaning - this contains the BGP global default configuration overrides.
+- Resource with the name "node.<nodename>" has the specific meaning that this contains the node-specific overrides, and will be applied to the node <nodename>.  When deleting a node the BGPConfiguration resource associated with the node will also be deleted.
```

Comment:
I am not sure, but I think that there are some global defaults, which can be overriden on a global basis via resources named `default`, and then in turn, both of these can be further overriden on a node-specific basis. If am correct, then I would suggest changing this first sentence a little bit as follows, "To specify node-specific overrides to global settings, use the name `node.<nodename>`."

Reply:
let's chat about this offline, I think we have a different understanding of what the global and node specific resources mean